app/models/SuperModel.coffee

```diff
@@ -132,9 +140,13 @@ module.exports = class SuperModel extends Backbone.Model
   # Tracking resources being loaded for this supermodel
 
   finished: ->
-    return @progress is 1.0 or not @denom
+    return (@progress is 1.0) or (not @denom) or @failed 
```

Comment:
Yes, the SuperModel is permanently finished and in an error state. Hadn't thought about passing supermodels to new views though. But looking at the code, it should be fine because [when a supermodel is passed to a view, it's cloned](https://github.com/codecombat/codecombat/blob/master/app/views/core/CocoView.coffee#L35-L38). Also, links that go to a view with a supermodel passed will be pretty scarce when an error takes over the view. Is there a particular use case that comes to mind that should be checked?